services/backend/src/recommendation_system/recommendation_flow/model_prediction/AlphaModel.py

```diff
@@ -1,43 +1,35 @@
 import random
+# from src import db
 import pandas as pd 
 import numpy as np
 from collections import defaultdict
+# from sqlalchemy.sql import text
 import os
 import pickle
 import sys
 import subprocess
 from .AbstractModel import AbstractModel
-
-def try_load_model(fn):
-    try:
-        with open(fn, 'rb') as f:
-            return pickle.load(f)
-    except:
-        pass
-
-dic_id_style = try_load_model('/usr/src/app/src/alpha/content_artist_style_dic.pickle')
-
-try:
-    subprocess.check_call([
-        sys.executable, 
-        "-m", 
-        "pip", 
-        "install", 
-        "tensorflow",
-        "keras",
-        "tensorflow_decision_forests"
-    ])
-    GBDT_model = try_load_model('/usr/src/app/src/alpha/gbdt_model_v3.pickle')
-    prep_dic = try_load_model('/usr/src/app/src/alpha/prediction_prep_dic.pickle')
-    dic_id_embed = try_load_model('/usr/src/app/src/alpha/dic_id_to_embedding.pickle')
-except:
-    pass
+from src.data_structures.approximate_nearest_neighbor import ann_with_offset, get_embedding
 
 
 class AlphaModel(AbstractModel):
     def predict_probabilities(self, content_ids, user_id, seed=None, **kwargs):
+        
         try:
+            print('try use tfdf')
+        #     # import tensorflow_decision_forests as tfdf
+            subprocess.check_call([sys.executable, "-m", "pip", "install", "tensorflow_decision_forests"])
             import tensorflow_decision_forests as tfdf
+
+            with open('/usr/src/app/src/alpha/gbdt_model_v3.pickle', 'rb') as f:
+                GBDT_model = pickle.load(f)
+            with open('/usr/src/app/src/alpha/content_artist_style_dic.pickle', 'rb') as f:
+                dic_id_style = pickle.load(f)
+            with open('/usr/src/app/src/alpha/prediction_prep_dic.pickle', 'rb') as f:
+                prep_dic = pickle.load(f)
+        #    with open('/usr/src/app/src/alpha/dic_id_to_embedding.pickle', 'rb') as f:
+        #        dic_id_embed = pickle.load(f)
+
```

Comment:
are you sure this is what you want? I think my code will be much much faster as it isn't loading these models each time and only installing the package once